src/services/SendEmailService.ts

```diff
@@ -128,30 +146,33 @@ export class SendEmailService {
     			this.logger.error("sendEmail - GOV UK Notify threw an error");
 
     			if (err.response) {
-    				// err.response.data.status_code 	err.response.data.errors
     				this.logger.error(`GOV UK Notify error ${SendEmailService.name}`, {
     					statusCode: err.response.data.status_code,
     					errors: err.response.data.errors,
     				});
     			}
 
-    			const appError: any = this.govNotifyErrorMapper.map(err);
+				const appError: any = this.govNotifyErrorMapper.map(err.response.data.status_code, err.response.data.errors[0].message);
 
-    			if (appError.obj!.shouldThrow) {
-    				this.logger.error("sendEmail - Mapped error", SendEmailService.name, appError.message);
-    				throw appError;
-    			} else {
-    				this.logger.error(`sendEmail - Mapped error ${SendEmailService.name}`, { appError });
-    				this.logger.error(`sendEmail - Retrying to send the email. Sleeping for ${this.environmentVariables.backoffPeriod()} ms ${SendEmailService.name} ${new Date().toISOString()}`, { retryCount });
-    				await sleep(this.environmentVariables.backoffPeriod());
-    			}
+				if (appError.obj!.shouldRetry) {
+					this.logger.error(`sendEmail - Mapped error ${SendEmailService.name}`, { appError });
+					if (retryCount < this.environmentVariables.maxRetries() + 1) {
```

Comment:
if (retryCount <= this.environmentVariables.maxRetries() + 1)
Also think this will now retry once only for Gov notify failures, needs a while loop I think to retry a number of times with sleep and separated into a new method pls?